packages/plugin-dotnet-scripting/index.js

```diff
@@ -0,0 +1,31 @@
+import {
+  FAKE_NUGET,
+  NET_SCRIPT_FILE,
+  NET_SCRIPT_NUGET,
+} from '@octolinker/helper-grammar-regex-collection';
+import nugetResolver from '@octolinker/resolver-nuget';
+import relativeFile from '@octolinker/resolver-relative-file';
+
+export default {
+  name: 'DotNetScripting',
+
+  resolve(path, [target]) {
+    // this reference doesn't resolve to anything
+    if (target === './.fake/build.fsx/intellisense.fsx') {
+      return [];
+    }
+
+    return [relativeFile({ path, target }), nugetResolver({ target })];
```

Comment:
The signature of the `resolve` handler is `resolve(path, target, meta, regExp)`. 

Side note: Meta defaults to an empty object and is used in the context of [plugin-npm-manifest](https://github.com/OctoLinker/OctoLinker/blob/main/packages/plugin-npm-manifest/index.js#L19).

What I want to talk about is the fourth argument `regExp` which is the matching RegExp from line 29. In my PHP PR I make use of this to use different resolvers https://github.com/OctoLinker/OctoLinker/blob/87a137aa312f2a6e4b425f2e599744e2fa4f61f5/packages/plugin-php/index.js#L7-L13 maybe we can do something similar here. This allows use to be more specific and does reduce unnecessary look up in the OctoLinker API.

Reply:
That's awesome, I'll have to revisit some of the css & dotnet things I was working on because I'm pretty sure I'll be able to simplify them by checking the regex.